build/print.go

```diff
@@ -334,6 +334,10 @@ var opPrec = map[string]int{
 	"//":     precMultiply,
 	"%":      precMultiply,
 	"|":      precMultiply,
+	"&":      precMultiply,
+	"^":      precMultiply,
+	"<<":     precMultiply,
+	">>":     precMultiply,
```

Comment:
This doens't seem to match 
https://docs.python.org/3/reference/expressions.html#operator-precedence or https://github.com/google/starlark-go/blob/master/doc/spec.md#binary-operators

Reply:
Fixed.